src/pudl/metadata/resources/ferc1.py

```diff
@@ -135,6 +135,24 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "electric_oandm_ferc1": {
+        "description": "Operating and maintenance costs associated with producing electricty.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "report_year",
+                "annual_oandm_expense",
+                "oandm_expense_type",
```

Comment:
could this just be `expense` and `expense_type`? given that they are in and o&m table, the oandm feels duplicative to me.

Reply:
Sure!

Reply:
Done